### PR TITLE
Add description of different type of repositories contained in the GitHub organization

### PR DESCRIPTION
**Pull request by traversaro**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
 
 If you want to know more about us:
 * Visit our website at [**🌐 https://ami.iit.it/**](https://ami.iit.it/)
 * Follow us on Instagram https://www.instagram.com/artificialmechintelligence/
-* Follow us on Twitter https://twitter.com/ami_iit 
+* Follow us on Twitter https://twitter.com/ami_iit
+
+
+This GitHub organization contains different types of repositories:
+* **Reusable software** libraries that make algorithms developed in AMI, developed in C++, Python or MATLAB available to the research community and ready to be used. Examples:
```
Comment (lrapetti) [inline profile/README.md]: Are you planning to complete this part with examples or is it a leftover?

File: profile/README.md [hunk 2]
```diff
@@ -1,14 +1,24 @@
...
 If you want to know more about us:
 * Visit our website at [**🌐 https://ami.iit.it/**](https://ami.iit.it/)
 * Follow us on Instagram https://www.instagram.com/artificialmechintelligence/
-* Follow us on Twitter https://twitter.com/ami_iit 
+* Follow us on Twitter https://twitter.com/ami_iit
+
+
+This GitHub organization contains different types of repositories:
+* **Reusable software** libraries that make algorithms developed in AMI, developed in C++, Python or MATLAB available to the research community and ready to be used. Examples: 
+* **Reproducible publication software** for each paper published by AMI, to easily reproduce the results contained in the scientific papers written by AMI members. These repos can be identified by the `paper_` prefix
```
Comment (lrapetti) [inline profile/README.md]:
```suggestion
* **Reproducible publication software** for each paper published by AMI, to easily reproduce the results contained in the scientific papers written by AMI members. These repos can be identified by the `paper_` prefix.
```

File: profile/README.md [hunk 3]
```diff
@@ -1,14 +1,24 @@
...
 
 Our mission is to combine Artificial Intelligence with Mechanics to provide mankind the next generation of Humanoid Robots that will help us face the looming societal challenges.
 
 
 If you want to know more about us:
 * Visit our website at [**🌐 https://ami.iit.it/**](https://ami.iit.it/)
 * Follow us on Instagram https://www.instagram.com/artificialmechintelligence/
-* Follow us on Twitter https://twitter.com/ami_iit 
+* Follow us on Twitter https://twitter.com/ami_iit
+
```
Comment (lrapetti) [inline profile/README.md]:
```suggestion
* Follow us on Linkedin https://www.linkedin.com/company/ami-artificial-and-mechanical-intelligence
```

File: profile/README.md (hunk 1 above)
Comment (traversaro) [reply profile/README.md]: Leftover. I wanted to put this example, but then I tought people could ask me how I selected the examples I included. So I tought in a following PR I can just automatically create a list with the different repos (as done in https://github.com/robotology/.github/blob/main/.github/workflows/index-update.yml), and remove the examples here.

**Review by CarlottaSartore — APPROVED** [review]

**Pull request merged**